adoc/retail_chap_formulas.adoc

```diff
@@ -0,0 +1,209 @@
+This configuration allows for the branch server to provide DHCP, DNS, TFTP, PXE and FTP services to terminals, which are configured through {smr} formulas in the {susemgr} {webui}.
+
+
+.Procedure: Setting up a branch server with a dedicated LAN
+
+. In the {susemgr} {webui}, open the details page for the branch server, and navigate to the [guimenu]``Formulas`` tab.
+. In the [guimenu]``Branch Network`` section, set these parameters:
+* Keep [guimenu]``Dedicated NIC`` checked
+* In the [guimenu]``NIC`` field, enter the name of the network device that is connected to the internal LAN.
+* In the [guimenu]``IP`` field, enter the static IP address to be assigned to the branch server on the internal LAN.
+* In the [guimenu]``Netmask`` field, enter the network mask of the internal LAN.
+. Check [guimenu]``Enable Route`` if you want the branch server to route traffic from internal LAN to WAN.
+* Check [guimenu]``Enable NAT`` if you want the branch server to convert addresses from internal LAN to WAN.
+* Select the [guimenu]``bind`` DNS forwarder mode.
+* Check DNS forwarder fallback if you want to rely on an external DNS if the branch DNS fails.
+* Specify the working directory, and the directory owner and group.
+. Click btn:[Save] to save your changes.
+. Apply the highstate.
+
+
+
+=== Set up a branch server with a shared network
+
+In this configuration, the branch server has only one network interface card, which is used to connect to the {susemgr} server as well as the terminals.
+
+This configuration allows for the branch server to provide DNS, TFTP, PXE and FTP services to terminals, which are configured through {smr} formulas in the {susemgr} {webui}.
+Optionally, the branch server can also provide DHCP services in this configuration.
+
+[NOTE]
+====
+If DHCP services are not provided by the branch server, ensure that your external DHCP configuration is set correctly:
+* The [systemitem]``next-server`` option must point to the branch server for PXE boot to work
+* The [systemitem]``filename`` option must correctly identify the network boot program (by default, this is [path]``/boot/pxelinux``)
+* The [systemitem]``domain-name-servers`` option must point to the branch server for correct host name resolution
+====
+
+
+.Procedure: Setting up a branch server with a shared network
+
+. In the {susemgr} {webui}, open the details page for the branch server, and navigate to the [guimenu]``Formulas`` tab.
+. In the [guimenu]``Branch Network`` section, set these parameters:
+* Keep [guimenu]``Dedicated NIC`` unchecked
+* Select which services to enable on the branch server’s firewall.
+Ensure you include DNS, TFTP and FTP services.
+* Select the [guimenu]``bind`` DNS forwarder mode.
+* Check DNS forwarder fallback if you want to rely on an external DNS if the branch DNS fails.
+* Specify the working directory, and the directory owner and group.
+. Click btn:[Save] to save your changes.
+. Apply the highstate.
+
+
+
+[[retail.sect.formulas.image-sync]]
```

Comment:
I would skip image-sync formula completely. It does not have any GUI and usage is covered in [Synchronize images to the servers](https://github.com/SUSE/doc-susemanager/blob/develop/adoc/retail_chap_install.adoc#synchronize-images-to-the-servers)